.github/workflows/pull-request.yml

```
@@ -14,5 +14,19 @@ jobs:
           dotnet-version: 6.0.x
       - name: Install dependencies
         run: dotnet restore ./EasyCaching.sln
+        
       - name: Build
-        run: dotnet build ./EasyCaching.sln --configuration Release --no-restore
\ No newline at end of file
+        run: dotnet build ./EasyCaching.sln --configuration Release --no-restore 
+        
+      - name: Build docker for tests
+        run: |
+          docker build . --tag local/easy-caching:latest --target tests
+          docker-compose up -d
+      
+      - name: Run integration tests
```

```suggestion
      - name: Run tests
```
Там же все тесты, а не только интеграционные